m4_sessions/1801-inning/group_5.md

```diff
@@ -3,11 +3,27 @@ Group 5
 
 Group Members:
 
+David Ryan
+Jesse Packwood
+Dione Wilson
+Randy
+
 Name of Session:
 
+From Introvert to Influential
+
 Session Description (Why should students come to this session):
 
+Are you an introvert? Do you feel like people often don't remember you or value your opinions? This session is designed to help you break out of your shell to make better connections when meeting new people. You will have some tools you can apply to be more influential to others and how to handle various social situations you may find yourself in. 
+
 Session Outcomes (What will students learn/leave with?):
 
+Students should leave with more confidence and some useful skills/strategies on how to interact with others in a way that is impactful, professional, and makes them more likable.
+
 Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
 
+15 minutes intros
+15 minutes how could your intro have been more impactful? asking questions, being genuinely interested in others
+15 minutes identifying weaknesses, what holds us back from speaking out more?
```

Comment:
Possible resource: https://www.ted.com/talks/susan_cain_the_power_of_introverts/transcript
https://www.npr.org/2014/11/21/364148816/why-do-we-undervalue-introverts